model/src/main/java/de/danoeh/antennapod/model/feed/FeedPreferences.java

```diff
@@ -60,7 +60,7 @@ public static NewEpisodesAction fromCode(int code) {
     }
 
     public enum SkipSilence {
-        OFF(0), GLOBAL(1), AGGRESSIVE(2);
+        GLOBAL(0), OFF(1), MILD(2), MEDIUM(3), AGGRESSIVE(4);
```

Comment:
Please keep the IDs of the existing settings the same. Otherwise this PR changes the settings for existing users by interpreting the stored values differently.

Reply:
reverted